ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -567,6 +590,10 @@ extension BottomCommandingController: UITableViewDataSource {
 }
 
 extension BottomCommandingController: UITableViewDelegate {
+    public func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
```

Comment:
what is this for? not clear from the PR description

Reply:
UITableView adds an arbitrary 20px margin at the bottom after the last section. Doing this stops it from doing that and results in a more predictable table view sizing. I'll add a comment.

Reply:
so then shouldn't it be done for only last section?

Reply:
We don't have section footers, so what we return here does reflect reality 😄

Reply:
Sounds like a good case for a preconditionFailure then?

Reply:
Can you elaborate? This is where the delegate returns footer sizes. The table view calls this even when there are no section footers.

Reply:
Ok that piece I did not know - if a table view calls this even when there are no footers that seems like an Apple bug we should file?